knowage-vue/src/modules/documentExecution/documentDetails/dialogs/olapDesignerDialog/DocumentDetailOlapDesignerDialog.vue

```diff
@@ -0,0 +1,168 @@
+            hiddenFormData.set('document', decodeURIComponent('' + this.document.id))
+            hiddenFormData.set('user_id', decodeURIComponent(uniqueID))
+            hiddenFormData.set('DOCUMENT_LABEL', decodeURIComponent(this.document.label))
+            hiddenFormData.set('SBI_COUNTRY', decodeURIComponent(country))
+            hiddenFormData.set('DOCUMENT_AUTHOR', decodeURIComponent(this.document.creationUser))
+            hiddenFormData.set('DOCUMENT_DESCRIPTION', decodeURIComponent(this.document.description))
+            hiddenFormData.set('SBI_LANGUAGE', decodeURIComponent(language))
+            hiddenFormData.set('DOCUMENT_NAME', decodeURIComponent(this.document.name))
+            hiddenFormData.set('NEW_SESSION', decodeURIComponent('TRUE'))
+            hiddenFormData.set('SBI_EXECUTION_ID', decodeURIComponent(this.sbiExecutionId))
+            hiddenFormData.set('EDIT_MODE', decodeURIComponent('null'))
+            hiddenFormData.set('timereloadurl', decodeURIComponent('' + new Date().getTime()))
+            hiddenFormData.set('ENGINE', 'knowageolapengine')
+
+            this.$store.commit('setLoading', true)
+            await this.$http.post(process.env.VUE_APP_OLAP_PATH + `olap/startolap/edit`, hiddenFormData, { headers: { Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9' } }).then(() => {})
+            this.$store.commit('setLoading', false)
+        },
+        async loadMondrianSchemaResources() {
+            this.$store.commit('setLoading', true)
+            await this.$http.get(process.env.VUE_APP_RESTFUL_SERVICES_PATH + `2.0/mondrianSchemasResource`).then((response: AxiosResponse<any>) => (this.mondrianSchemas = response.data))
+            this.$store.commit('setLoading', false)
+        },
+        closeDialog() {
+            this.$emit('close')
+            this.type = 'xmla'
+            this.xmlModel = { address: '', parameters: [] }
+            this.mondrianModel = {} as iMondrianTemplate
+        },
+        async start() {
+            const postData = this.type === 'xmla' ? { ...this.xmlModel } : { ...this.mondrianModel }
+            this.$store.commit('setLoading', true)
+            await this.$http
+                .post(process.env.VUE_APP_OLAP_PATH + `1.0/designer/cubes?SBI_EXECUTION_ID=${this.sbiExecutionId}`, postData, { headers: { Accept: 'application/json, text/plain, */*' } })
+                .then(() => {
+                    this.$emit('designerStarted', { ...this.selectedDocument, sbiExecutionId: this.sbiExecutionId, reference: this.mondrianModel?.mondrianSchema, artifactId: this.mondrianModel.mondrianSchemaId })
+                })
+                .catch(() => {})
+            this.$store.commit('setLoading', false)
+        }
+    }
+})
+</script>
+
+<style lang="scss">
+.full-screen-dialog.p-dialog {
+    max-height: 100%;
+    height: 100vh;
+    width: calc(100vw - #{54px});
```

Comment:
if this one is intended to occupy the main menu width please use var(--kn-mainmenu-width) instead of 54px

Reply:
Fixed it